<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Robert" failures="1" tests="8" name="Test1" time="0.096" errors="0" timestamp="2024-07-24T01:35:12 IST">
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod1" time="0.014"/>
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod2" time="0.008">
    <failure type="java.lang.AssertionError" message="expected [11] but found [10]">
      <![CDATA[java.lang.AssertionError: expected [11] but found [10]
at TestNGStudy.testNGConcepts.ITestListenerExample.testMethod2(ITestListenerExample.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testMethod2 -->
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod3" time="0.003">
    <skipped/>
  </testcase> <!-- testMethod3 -->
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod4" time="0.002"/>
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod4" time="0.002"/>
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod4" time="0.001"/>
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod4" time="0.001"/>
  <testcase classname="TestNGStudy.testNGConcepts.ITestListenerExample" name="testMethod4" time="0.002"/>
</testsuite> <!-- Test1 -->
